Duplicati/CommandLine/RecoveryTool/FileIndex.cs

```diff
@@ -0,0 +1,246 @@
+﻿//  Copyright (C) 2015, The Duplicati Team

```

Comment:
This is a rename of the original file-based `Index` class.  The only modification was to move the call to `SortFile` [from the](https://github.com/duplicati/duplicati/pull/4061/files#diff-b389042e9144a8f9c477f231f51c2427L61) `Restore` class [to the end](https://github.com/duplicati/duplicati/pull/4061/files#diff-c02ab5ee0afabad37142d2d16e7851bfR135) of the `FileIndex.Run` method.